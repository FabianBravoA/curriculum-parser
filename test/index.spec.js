'use strict';


const CourseParser = require('../');


describe('CourseParser(paths, opts, cb)', () => {

  it('should yield { courses: {}, log: [] } for empty paths', (done) => {
    CourseParser([], {}, (err, data) => {
      expect(err).toBe(null);
      expect(data).toEqual({ courses: {}, log: [] });
      done();
    });
  });

  it('should yield error when path doesnt exist', (done) => {
    CourseParser(['foo'], {}, (err, data) => {
      expect(err).toMatchSnapshot();
      expect(data).toBe(undefined);
      done();
    });
  });

});
