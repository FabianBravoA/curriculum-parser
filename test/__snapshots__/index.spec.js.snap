// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CourseParser(paths, opts, cb) should yield error when path doesnt exist 1`] = `[Error: ENOENT: no such file or directory, open '/Users/lupo/work/laboratoria/course-parser/foo']`;
